Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1158,6 +1158,14 @@ The current Cilium kube-proxy replacement mode can also be introspected through
     $ kubectl exec -it -n kube-system cilium-xxxxx -- cilium status | grep KubeProxyReplacement
     KubeProxyReplacement:   Strict	[eth0 (DR)]
 
+Graceful Termination
+*********************
```

Comment:
tiny nit: I think one '*' too much, not sure if it still keeps rendering docs then.